mlir/test/Dialect/Affine/invalid.mlir

```diff
@@ -563,3 +563,17 @@ func.func @no_upper_bound() {
   }
   return
 }
+
+// -----
+
+func.func @invalid_symbol() {
```

Comment:
Is this the right design? Or should the verifier of this op call `legalizeDemotedDims` and make the op as it is expected to be?

Reply:
A verifier should only verify the op the way it is! It can't transform it!